vs/sdl/Makefile.minimal

```diff
@@ -1,9 +1,11 @@
 # Makefile to build the SDL library
 
-INCLUDE = -I./include
-CFLAGS  = -g -O2 $(INCLUDE)
-AR	= ar
-RANLIB	= ranlib
+INCLUDE += -I./include
```

Comment:
This should be fixed [upstream](https://github.com/libsdl-org/SDL/blob/main/Makefile.minimal).

Reply:
I think SDL1 is not supported anymore upstream?
Does that mean it's OK to fix it directly here?

Reply:
I guess it does!